content/blog/2022-06-29-june-community-gems.md

```diff
@@ -0,0 +1,215 @@
+---
+title: June '22 Community Gems
+date: 2022-06-29
+description: >
+  A roundup of technical Q&A's from the DVC and CML communities. This month:
+  working with the DVC cache, DVC data and remotes, using DVC programmatically,
+  and more.
+descriptionLong: >
+  A roundup of technical Q&A's from the DVC and CML communities. This month:
+  working with the DVC cache, DVC data and remotes, using DVC programmatically,
+  and more.
+picture: 2022-06-29/june-community-gems.png
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/may-22-community-gems/1184
+tags:
+  - DVC Remotes
+  - Pipelines
+  - CML
+  - GCP
+  - Community
+---
+
+## [Is there something like `git add-commit -m "some message"` for DVC?](https://discord.com/channels/485586884165107732/563406153334128681/981498675689828362)
+
+Thanks for the question @Ramnath T!
+
+While there isn't a single command to handle this, you can get similar behavior
+with DVC with an alias, like the following:
+
+```dvc
+$ alias dvc-add-commit='dvc add . & dvc commit'
```

Comment:
I don't think this alias works (`dvc add .` will fail) and it wouldn't make much sense anyway. You wouldn't want to `dvc add` your whole workspace, and `dvc commit` would be redundant if you did. Just `dvc commit` should really be enough here.

Reply:
It does run in a project I have set up, but I see what you mean about it not making sense to add the whole workspace. I just took out the alias tidbit and went back to what I had originally.